sigs.yaml

```diff
@@ -3111,22 +3111,22 @@ workinggroups:
   label: multitenancy
   leadership:
     chairs:
-    - github: srampal
-      name: Sanjeev Rampal
-      company: Cisco
-    - github: tashimi
+     - github: tashimi
```

Comment:
verify script is unhappy with this line... is indenting right?

Reply:
done!